tools/docker-compose/README.md

```diff
@@ -0,0 +1,20 @@
+# Docker Compose Tools
+
+[Docker Compose](https://github.com/docker/compose/blob/master/README.md) is a tool for defining and running multi-container Docker applications. With Compose, you use a Compose file to configure your application's services. Then, using a single command, you create and start all the services from your configuration.
```

Comment:
nit: Wrap at 80 columns please.

Thank you for this readme and intro descriptions. It's helpful. Since you've copied the text verbatim from compose's readme, please put it in quotes.